DQM/TrackerMonitorTrack/src/MonitorTrackResiduals.cc

```diff
@@ -38,6 +38,11 @@ void MonitorTrackResidualsBase<pixel_or_strip>::bookHistograms(DQMStore::IBooker
     m_cacheID_ = cacheID;
     this->createMEs( ibooker , iSetup);
   }
+  std::string topFolderName_ = "SiStrip";
+  SiStripFolderOrganizer folder_organizer;
+  folder_organizer.setSiStripFolderName(topFolderName_);
+  tkhisto_ResidualsMean = new TkHistoMap(ibooker , topFolderName_ ,"TkHMap_ResidualsMean", 0.0,true);
```

Comment:
Hi @aspiezia,
forget about deleting at the end job and do what @davidlange6 suggested - use unique_ptrs.

Modify https://github.com/aspiezia/cmssw/blob/a0a714e6c65870a27ee5a346d1d0e75367573276/DQM/SiStripMonitorTrack/interface/SiStripMonitorTrack.h#L131 and change TkHistoMap *x to std::unique_ptr<TkHistoMap> x;